Makefile

```diff
@@ -79,11 +79,10 @@ endif
 
 # Download protoc plugins
 protoc-plugins:
-	@go install \
-		github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway \
-		github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2 \
-		google.golang.org/protobuf/cmd/protoc-gen-go \
-		google.golang.org/grpc/cmd/protoc-gen-go-grpc
+	@go install github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway@v2.15.0
+	@go install github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2@v2.15.0
+	@go install google.golang.org/protobuf/cmd/protoc-gen-go@b0a9446
```

Comment:
`progoc-gen-go` isn't correctly tagged in `google.golang.org/protobuf`: the tag `cmd/protoc-gen-go/v1.28.1` is missing, `b0a9446` is the commit tagged with that version.